ietf-wdm-tunnel.yang

```diff
@@ -426,6 +522,31 @@ module ietf-wdm-tunnel {
     }
   }
 
+  grouping digital-layer-constraint {
+    description
+      "Grouping for digital layer constraints";
+
+    container digital-constraint {
+      description
+        "Digital layer constraints.";
+
+      leaf digital-termination {
```

Comment:
Just a question for clarification: the termination layer is a single value for tunnel. But where would be defined the possible layer of termination in case of encapsulation ? e.g. ODU-->OTU